.gitignore

```diff
@@ -1,10 +1,223 @@
+# Created by https://www.toptal.com/developers/gitignore/api/android,androidstudio,gradle,kotlin,java
+# Edit at https://www.toptal.com/developers/gitignore?templates=android,androidstudio,gradle,kotlin,java
+
+### Android ###
+# Gradle files
+.gradle/
+build/
```

Comment:
@Animansoubi Please take a look at these changes, This is a sample gitignore configuration for an Android project that has been made during these years by the experience. Then It's good to have this as the gitignore in our project.

Reply:
However, some of them are extra and do not fit our needs. But no harm in keeping them.